docs/content/en/docs/getting-started/developer/_index.md

```diff
@@ -0,0 +1,215 @@
+k8ssandra-dc1-service                       ClusterIP   None             <none>        9042/TCP,9142/TCP,8080/TCP,9103/TCP,9160/TCP            21h
+k8ssandra-dc1-stargate-service              ClusterIP   10.106.70.148    <none>        8080/TCP,8081/TCP,8082/TCP,8084/TCP,8085/TCP,9042/TCP   21h
+k8ssandra-grafana                           ClusterIP   10.96.120.157    <none>        80/TCP                                                  21h
+k8ssandra-kube-prometheus-operator          ClusterIP   10.97.21.175     <none>        443/TCP                                                 21h
+k8ssandra-kube-prometheus-prometheus        ClusterIP   10.111.184.111   <none>        9090/TCP                                                21h
+k8ssandra-reaper-k8ssandra-reaper-service   ClusterIP   10.104.46.103    <none>        8080/TCP                                                21h
+k8ssandra-seed-service                      ClusterIP   None             <none>        <none>                                                  21h
+kubernetes                                  ClusterIP   10.96.0.1        <none>        443/TCP                                                 21h
+prometheus-operated                         ClusterIP   None             <none>        9090/TCP                                                2
+```
+
+In the output above, the pod of interest is:
+
+* **k8ssandra-dc1-stargate-service**: The K8ssandra Stargate where the name is a combination of the K8ssandra cluster name you specified during the Helm install, `k8ssandra`, the datacenter name, `dc1` and the postfix, `-service`. This service listens on the ports:
+  * **8080/TCP**: GraphQL interface
+  * **8081/TCP**: REST authorization service for generating tokens
+  * **8082/TCP**: REST interface
+  * **8084/TCP**: Health check (/healthcheck)
+  * **8085/TCP**: Metrics (/metrics)
+  * **9042/TCP**: CQL service
+
+Those are the ports we'll need to forward for CQLSH and Stargate access.
+
+To configure port forwarding:
+
+1. Open a new terminal.
+
+2. Run the `kubectl port-forward` command:
+
+    ```bash
+    kubectl port-forward svc/k8ssandra-dc1-stargate-service 8080 8081 8082 8084 8085 9042
+    Forwarding from 127.0.0.1:8080 -> 8080
+    Forwarding from [::1]:8080 -> 8080
+    Forwarding from 127.0.0.1:8081 -> 8081
+    Forwarding from [::1]:8081 -> 8081
+    Forwarding from 127.0.0.1:8082 -> 8082
+    Forwarding from [::1]:8082 -> 8082
+    Forwarding from 127.0.0.1:8084 -> 8084
+    Forwarding from [::1]:8084 -> 8084
+    Forwarding from 127.0.0.1:8085 -> 8085
+    Forwarding from [::1]:8085 -> 8085
+    Forwarding from 127.0.0.1:9042 -> 9042
+    Forwarding from [::1]:9042 -> 9042
+    ```
+
+3. Leave the terminal running in the background.
+
+{{% alert title="Important" color="warning" %}}
+If you close the terminal for any reason, you'll shut down the port forwarding service.
```

Comment:
Why not start this command in the background? And then something like [this](https://stackoverflow.com/questions/37288500/how-to-undo-a-kubectl-port-forward) later on to clean up?

Reply:
Just what I needed for the site engineer section that will need a couple of these running. Thanks!